### PR TITLE
Making avro fallback to not using namespace when parsing sub records.

### PR DESCRIPTION
**Pull request by anyman**
This will allow the addition of a namespace to schemas while still being able to parse old data

### CODE REVIEW

File: test/avro_tests.py [hunk 1]
```diff
@@ -344,3 +344,25 @@ class ValidRecord(pyschema.Record):
...
 
         self.assertRaises(ParseError, lambda: pyschema_extensions.avro.loads(line, schema=ValidRecord))
+
+
+class TestNamespaceMigration(TestCase):
+    def test_old_data(self):
+        """
+        Test that we are able to read data created without namespace after namespace is added.
+        """
+        class NamespacedSubRecord(Record):
```
Comment (freider) [inline test/avro_tests.py:358]: Probably best to add a @no_auto_store() here to avoid cluttering the testing namespace

**Comment by freider** [comment]
One comment. Otherwise lgtm

File: test/avro_tests.py (hunk 1 above)
Comment (anyman) [inline test/avro_tests.py:358]: I need to do it in order to test the parsing when not including schema in the call to pyschema_extensiions.avro.loads

That test might not be so relevant however

**Comment by anyman** [comment]
Updated to use a test schema_store instead of the auto_store

**Pull request merged**